cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,11 +91,13 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_NAME = "cruise-control-metrics-and-logging";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
+    protected static final String CRUISE_CONTROL_CONFIG_VOLUME_NAME = "config";
+    protected static final String CRUISE_CONTROL_SERVER_CONFIG_FILENAME = "server-config";
+    protected static final String CRUISE_CONTROL_CAPACITY_CONFIG_FILENAME = "capacity-config";
+    protected static final String CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
     protected static final String API_AUTH_CONFIG_VOLUME_NAME = "api-auth-config";
     protected static final String API_AUTH_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/api-auth-config/";
-
+    protected static final String CAPACITY_CONFIG_FILE = CONFIG_VOLUME_MOUNT + CRUISE_CONTROL_CAPACITY_CONFIG_FILENAME;
```

Comment:
I think this is confusing as you already have one similarly sounding variable. I also wonder if you really need a constant for it. Just use the original two constants in the one place (+ test) where you seem to use this.

Reply:
I agree, this file path seems to be only used in CruiseControlConfiguration at the moment.

Reply:
Removed